templates/cli/lib/commands/pull.js.twig

```diff
@@ -22,6 +22,8 @@ const pullProject = async () => {
         })
 
         localConfig.setProject(response.$id, response.name);
+        localConfig.setProjectSettings(response);
```

Comment:
Lets just add settings as an optional argument to the setProject() function itself.